node/packages/sdk/package.json

```diff
@@ -0,0 +1,55 @@
+{
+  "name": "@serverless/sdk",
```

Comment:
@medikoo Looks like we used the `@serverless/sdk` name already years ago, https://www.npmjs.com/package/@serverless/sdk. 

We will need to change this to something else in order to publish it.

Reply:
@Mmarzex yes, I'm aware of that. It's an old project that's not used at all and left abandoned  5 years ago.

It's also our package, and we have rights to publish new versions. I plan to prepare a v0.1.0 release after this PR is taken